### PR TITLE
Remove proof of signature

### PR DESCRIPTION
**Pull request by nholland94**

### CODE REVIEW

**Comment by nholland94** [comment]
This PR depends on https://github.com/CodaProtocol/coda/pull/2300

**Comment by nholland94** [comment]
Woops, closed by accident.

File: src/lib/consensus/consensus.ml [hunk 1]
```diff
@@ -5,11 +5,6 @@ module Constants = Constants
 include Intf
 
 [%%if
```
Comment (imeckler) [inline src/lib/consensus/consensus.ml]: Delete?

Comment (nholland94) [reply src/lib/consensus/consensus.ml]: I left this to still provide an error if there is a configuration with a `consensus_mechanism` value other than `"proof_of_stake"` (note the `[%%error ...]` in the else case).

**Pull request merged**